src/scrapy_settings_log/__init__.py

```diff
@@ -51,6 +51,16 @@ def spider_closed(self, spider):
         regex = settings.get("SETTINGS_LOGGING_REGEX")
         if regex is not None:
             settings = {k: v for k, v in settings.items() if re.search(regex, k)}
+        if settings.get("MASKED_SENSITIVE_SETTINGS_ENABLED", True):
```

Comment:
**todo:** We should use `settings.getbool()` here, which enables parsing specific strings (`"1"`, `"True"`) as booleans. Setting values are passed as strings when they are defined via command line or ScrapyCloud, so it's nice to support them.